### PR TITLE
device-manager: add new default action Ping Device.

### PR DESCRIPTION
**Pull request by element-82**
Add new default action "Ping Device".  This action is intended to test
the transport from cloud to agent and return a timestamp in UTC.  This
action is suitable to be called from a trigger for testing uptime on
devices.  This action is also a diagnostic utility for testing
action sanity on the device manager.

Signed-off-by: Paul Barrette <paulbarrette@gmail.com>

### CODE REVIEW

File: iot_build.h.in [hunk 1]
```diff
@@ -120,6 +120,9 @@
 /** @brief Websocket long description */
 #define IOT_RELAY_DESCRIPTION          "@IOT_RELAY_DESCRIPTION@"
 
+/** @brief Default state of action ping*/
+#define IOT_DEFAULT_ENABLE_PING        @IOT_DEFAULT_ENABLE_PING@
```
Comment (lt-holman) [inline iot_build.h.in]: 1) keep in same order as other actions
2) keep alignment the same as other actions.

File: src/device-manager/device_manager_main.c [hunk 2]
```diff
@@ -1646,6 +1684,43 @@ iot_status_t on_action_agent_quit(
...
+#	define DM_TIMESTAMP_LEN    25u
+
+	if ( device_manager && request )
+	{
+		iot_t *const iot_lib = device_manager->iot_lib;
+		if ( iot_lib )
+		{
+			size_t out_len;
+			iot_timestamp_t ts;
+			char ts_str[32];
```
[the remark this answers is not in the document]
Comment (element-82) [reply src/device-manager/device_manager_main.c]: Ack.

File: src/device-manager/device_manager_main.h [hunk 3]
```diff
@@ -58,6 +58,7 @@ enum device_maanger_config_idx
 	DEVICE_MANAGER_IDX_FIRST = 0,
 	DEVICE_MANAGER_IDX_AGENT_RESET = 0,
 	DEVICE_MANAGER_IDX_AGENT_QUIT,
+	DEVICE_MANAGER_IDX_PING,
```
[the remark this answers is not in the document]
Comment (element-82) [reply src/device-manager/device_manager_main.h]: Ack

File: iot_build.h.in (hunk 1 above)
Comment (element-82) [reply iot_build.h.in]: Ack

**Pull request merged**